dynamic_graph_service/python/coordinator/http_service.py

```diff
@@ -83,15 +104,29 @@ def do_GET(self):
       self.send_response(200)
       self.end_headers()
       self.wfile.write(bytes(dl_init_json_str, "UTF-8"))
+    elif self.path.startswith("/admin/barrier-status"):
+      self.send_response(200)
+      self.end_headers()
+      param = self.path.split("?")[1]
```

Comment:
Better to catching the exception of index out of range

Reply:
using urllib.parse